javascript/javascript1/week1/housey.js

```diff
@@ -0,0 +1,12 @@
+let volumeInMeters =800;
+let houseCost = 2500000;
+let gardenSizeInM2 =100;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```

Comment:
We should declare this variable with the right keyword otherwise we get an error. Here we can have a `const` since we won't need to reassign this value